intergov/processors/multichannel_router/__init__.py

```diff
@@ -219,7 +202,8 @@ def __next__(self):
                     str(e)
                 )
                 self.outbox_repo.patch(pg_msg.id, {'status': 'rejected'})
-                time.sleep(random.randint(3, 10))
+                for i in range(random.randint(30, 100)):
+                    time.sleep(0.1)
```

Comment:
I bet there is a good reason for this, but it's not obvious to me. Why?

Reply:
https://stackoverflow.com/questions/5114292/break-interrupt-a-time-sleep-in-python

Reply:
in fact it doesn't make much difference except for some local development; and I'm not sure it will survive in the final code.

Reply:
Interesting - so that OS-level process management can work faster. No problem, was just curious (and, better than messing with threads)